assets/js/root.jsx

```diff
@@ -82,6 +82,7 @@ class Root extends React.Component {
                 localStorage["project2_session"] = JSON.stringify(resp.data);
                 let state1 = _.assign({}, this.state, { session: resp.data, error: null });
                 this.setState(state1);
+                window.location.href = "/users/" + this.state.session.user_id;
```

Comment:
ideally we would be able to use `props.history.push(...)` but think we would need to do some refractoring first, moving this into the `Header` component